tests/test_file_checkers.py

```diff
@@ -99,8 +98,12 @@ def test_reviewer_integration(package_path, check_runner):
     assert not (warning_asserts and all_warning_asserts), \
         "Only one warnings meta file is allowed"
 
-    assert_none = not (failure_asserts or all_failure_asserts
-                       or warning_asserts or all_warning_asserts)
+    assert_none = not (
+        failure_asserts or
```

Comment:
I prefer having the binary operator on the new line to more clearly show its relation to the above line. I believe there is an error code for this that goes both ways, so I'll see if I can enable this check for the future.